recipe/meta.yaml

```diff
@@ -80,6 +80,9 @@ source:
       - patches/0034-have-pyunicode-decodeunicodeescape.patch
       # MailCap CVE (CVE-2015-20107) was fixed in 3.9.16
       # - patches/0035-py39-mailcap-CVE-2015-20107.patch
+{% if '3.0' in openssl %}
```

Comment:
Maybe `{% if openssl.startswith('3.') %}` would be more reliable and future proof?

Reply:
This fails but there is a fix that is more explicit.

Reply:
from @cbouss :

{% if (openssl | string).startswith('3.0') %}